.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,86 @@
+                        print(
+                            f"❌ Error: '@ts-ignore' found in {file} "
+                            f"at line {line_num}"
+                        )
+                        logging.debug(
+                            f"Found @ts-ignore in line: {line.strip()}"
+                        )
+                        ts_ignore_found = True
+        except FileNotFoundError:
+            logging.warning(f"File not found: {file}")
+        except OSError as e:
+            logging.error(f"Could not read {file}: {e}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)